test/Libraries/RevitIntegrationTests/RegressionTests.cs

```diff
@@ -79,6 +82,155 @@ public void Regressions(RegressionTestData testData)
             }
         }
 
+        [Test]
+        [TestModel(@".\empty.rfa")]
+        public void NoUINodesLeftBehind()
+        {
```

Comment:
These 2 tests share very similar logic and could be condensed to a single test.  For clarity and debugging I kept them separate but let me know if you think it is unnecessary.

Reply:
👍